src/python/pants/option/global_options.py

```diff
@@ -286,8 +286,10 @@ def register_bootstrap_options(cls, register):
                   'The `--pants-distdir` and `--pants-workdir` locations are inherently ignored.')
     register('--glob-expansion-failure', advanced=True,
              default=GlobMatchErrorBehavior.warn, type=GlobMatchErrorBehavior,
-             help="Raise an exception if any targets declaring source files "
-                  "fail to match any glob provided in the 'sources' argument.")
+             help="What to do when any filesystem specs cannot be found or any files in a target's "
```

Comment:
How about: "What to do when files specified on the command line or in BUILD files can't be found...". I'm not sure a "filesystem spec" is or should be user facing verbage, and the sources field seems needlessly specific. BUILD files might incorporate file globs in other ways.

Reply:
Much better! I agree with both things, including not referring to `filesystem specs` explicitly to users. I can't think of any tools like Pex, Pytest, or Black ever using a term like that. They're simply files.

Thanks!